### PR TITLE
Added transition on testimonial cards.

### PR DESCRIPTION
**Pull request by supzi-del**
## What is the change?
Added a transition duration of 0.5s when hovered upon testimonial card divs.

## Related issue?
close: #1000 

## Screen recording 

https://user-images.githubusercontent.com/78655439/121645080-6b1cbe00-cab1-11eb-9e50-f93d0217f15b.mp4

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [] Did you lint your code before making the Pull Request?
- [] Have you made corresponding changes to the documentation?
- [] Your submission doesn't break any existing feature.
- [] Have you tested the code before submission?

### CODE REVIEW

**Review by Ayush0730 — APPROVED** [review]

File: public/css/styles.css [hunk 1]
```diff
@@ -1446,6 +1446,7 @@ div.scroll {
 }
 .testimonial-card-div {
 	width: 60vw;
+	transition: all 0.5s;
```
Comment (Pranav016) [inline public/css/styles.css:1449]: Isn't the animation on hover so I don't think adding `transition` property to a div would affect that.

**Review by Pranav016 — CHANGES_REQUESTED** [review]
- Usually I'd advice adding a screenshot or a screen-recording(in this case) to the PR description to show that the changes actually work.
- Make PR from a new branch instead of master

File: public/css/styles.css (hunk 1 above)
Comment (supzi-del) [reply public/css/styles.css:1449]: I have tested it and only then made a PR. Please can you check the screen recording I have updated?

Comment (supzi-del) [reply public/css/styles.css:1449]: Should I make a PR to the "deploy" branch?

Comment (Pranav016) [reply public/css/styles.css:1449]: > Should I make a PR to the "deploy" branch?

Next time you can make PR from a new branch to master of ALPHAVIO/WordNook. Just a good practice.

**Pull request merged**